src/StackAdmin/Azs.KeyVault.Admin/Module/Azs.KeyVault.Admin/Azs.Keyvault.Admin.psd1

```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License. See License.txt in the project root for license
     RootModule        = 'Azs.KeyVault.Admin.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '0.1.0'
```

Comment:
The version is updated here, does it need to be updated for the other modules where you made changes?

Reply:
The other module versions are already updated with previous PRs